dice-game-ui-angular/src/app/nav-menu/nav-menu.component.html

```diff
@@ -1,6 +1,7 @@
-<mat-toolbar color='primary'>  
-  <mat-toolbar color='primary'>
+<mat-toolbar color="primary">  
+  <mat-toolbar color="primary">
     <span>Dice Game</span>
-    <button mat-flat-button color="primary" [routerLink]="['start']">Start Game</button>
+    <button mat-flat-button color="primary" [routerLink]="'start'">Start New Game</button>
```

Comment:
`routerLink` without brackets and then url without quotes.

Reply:
I used that because the docs tells to do that but, no problem, I'll change it.

Reply:
Fixed